piccolo/query/methods/alter.py

```diff
@@ -500,15 +501,18 @@ def drop_foreign_key_constraint(
         self, column: t.Union[str, ForeignKey]
     ) -> Alter:
         constraint_name = self._get_constraint_name(column=column)
-        return self.drop_constraint(constraint_name=constraint_name)
+        self._drop_constraint.append(
+            DropConstraint(constraint_name=constraint_name)
+        )
+        return self
 
     def add_foreign_key_constraint(
         self,
         column: t.Union[str, ForeignKey],
         referenced_table_name: str,
+        referenced_column_name: str,
         on_delete: t.Optional[OnDelete] = None,
         on_update: t.Optional[OnUpdate] = None,
-        referenced_column_name: str = "id",
```

Comment:
Why are we removing the default value?

Reply:
I changed this because with the default value I get an `'AddForeignKeyConstraint' object attribute 'referenced_column_name' is read-only` error. I don't know another way to solve it and I remove the default value and put the `referenced_column_name` in the slots. If there is a better way, please change it. Thanks.

Reply:
OK, fair enough - we should be passing in this value anyway.